.projenrc.js

```diff
@@ -114,7 +114,9 @@ project.gitignore.removePatterns("/dist/");
 // The two lines below makes git track "dist/go" and ignore all other folders
 // under "dist/"
 project.gitignore.addPatterns("dist/*");
-project.gitignore.include("!dist/go");
```

Comment:
`!` is not necessary, so removing it. This doesn't affect the generated `.gitignore`.